app/assets/stylesheets/normalize.scss

```diff
@@ -42,7 +42,7 @@ main,
 menu,
 nav,
 section,
```

Comment:
This trailing comma is invalid CSS =( Would we be able to comment it out too?

Reply:
probably?